kb/2023-09-01/vm_live_migration_policy_and_configuration.md

```diff
@@ -0,0 +1,325 @@
+---
+title: VM Live Migration Policy and Configuration
+description: Know how VM live migration works, the migration policies, how to tune the policies and check status
+slug: vm_live_migration_policy_and_configuration
+authors:
+  - name: Jian Wang
+    title: Staff Software Engineer
+    url: https://github.com/w13915984028
+    image_url: https://github.com/w13915984028.png
+tags: [harvester, virtual machine, VM, live migration, policy, strategy, configuration]
+hide_table_of_contents: false
+---
+
+# VM Live Migration
+
+In Harvester, the **VM Live Migration** is well supported by the UI. Please refer to [Harvester VM Live Migration](https://docs.harvesterhci.io/v1.1/vm/live-migration) for more details.
+
+The VM Live Migration process is finished smoothly in most cases. However, sometimes the migration may get stuck and not end as expected.
+
+This article dives into the VM Live Migration process in more detail. There are three main parts:
+
+- General Process of VM Live Migration
+- VM Live Migration Strategies
+- VM Live Migration Configurations
+
+Related issues:
+
+- [Migration should show the proper status and progress in the UI](https://github.com/harvester/harvester/issues/4352)
+- [VM Migration policy and status](https://github.com/harvester/harvester/issues/4376)
+
+## General Process of VM Live Migration
```

Comment:
The article looks good. Although we already list https://kubevirt.io/user-guide/operations/live_migration/ in reference, I'd suggest we have a note admonition at the beginning that state most content of the article is borrowed from kubevirt's doc.